packages/components/src/components/CodeEditor/languageSupport/squiggle.ts

```diff
@@ -103,6 +103,9 @@ export function squiggleLanguageSupport(project: SqProject) {
 
         FunctionName: t.function(t.variableName),
 
+        DecoratorName: t.variableName,
+        At: t.keyword,
```

Comment:
VS Code doesn't highlight `@`, but I think I like the green bold `@` symbol a bit more. Open to other suggestions.

Reply:
<img width="213" alt="Screenshot 2023-12-19 at 14 05 10" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/eb5117e6-83f7-44ca-8a97-a19e3878f358">

Reply:
This looks good to me